sdk/src/lib.rs

```diff
@@ -79,3 +80,6 @@ pub mod transport;
 extern crate serde_derive;
 pub extern crate bs58;
 extern crate log as logger;
+
+#[macro_use]
+extern crate solana_sdk_macro;
```

Comment:
Why the solana_sdk_macro crate?

Reply:
I just reused an existing crate which contained a procedural macro. Maybe you mean creating a new crate like `solana-serde-{frozen-abi, utils}` like `solana-clap-utils`?

Reply:
Well, I couldn't this because `Hash` and `AbiDigester` is inter-depended, iirc...